b2sdk/version.py

```diff
@@ -18,3 +23,178 @@
 PYTHON_VERSION = '.'.join(map(str, sys.version_info[:3]))  # something like: 2.7.11
 
 USER_AGENT = 'backblaze-b2/%s python/%s' % (VERSION, PYTHON_VERSION)
+
+
+class Deprecat:
+    def _get_verified_versions(self, tuple_, subject, from_=None, allow_future=False):
+        """
+        subject may be "function", "method", "argument", "class" etc
+        returns None if replacement is going to happen in the future, or a 3-tuple with replacemnt specification:
+            * name to replace to
+            * version where the change was introduced
+            * version where the support for the old name is going to be dropped
+        """
+        to_, version_changed, version_dropped = tuple_
+        assert len(version_changed) == len(version_dropped) == len(
```

Comment:
Will fail with multi-digit numbers in versions, like `0.1.12`?

Reply:
just looked at the Semantic Version doc, technically there is no limit on digits on the version string.
Maybe we should just check that the versions are digits separated by `.`?

Reply:
the only reasonable solution seems to be setuptools and `from pkg_resources import parse_version`